README.md

```diff
@@ -13,7 +13,9 @@ Display your deployed versions.
 
 ## QUICK START
 
-Using `make`.
+Using `make` to build and run the project(s).
+
+Install the `npn` dependencies for all projects, use `make install.local`
```

Comment:
I think here it should be `npm` :)

Reply:
Good spot 👍 . Typo by me, I will fix it now 😄